app/screens/post_options/index.js

```diff
@@ -94,6 +95,13 @@ export function mapStateToProps(state, ownProps) {
         canCopyText = true;
     }
 
+    const getReactionsForPostSelector = makeGetReactionsForPost();
```

Comment:
by adding this here, the reactions below will always return a different object as they are not correctly memoized. Try looking at how it is implemented in the `reactions/index.js` file where mapStateToProps is wrap in a makeMapStateToProps function to correctly use memoization

Reply:
@enahum Wondering if this is something we're able to cover on our end? @cardoso is busy in the near term per the comment below. 

Otherwise we could open the ticket for contributions if it's a substantial change